dxfeed/core/utils/handler.pyx

```diff
@@ -0,0 +1,107 @@
+from warnings import warn
+from dxfeed.core.utils.data_class import DequeWithLock as deque_wl
+import pandas as pd
+
+
+cdef class EventHandler:
+    """
+    Master class for user defined event handlers. `update` method should be considered as abstract.
+
+    Attributes
+    ----------
+    columns: list
+        After attaching listener the field contains one-word descriptors of the values in upcoming event the order
+        coincides
+    """
+    def __init__(self):
+        self.columns = list()
+
+    cdef void cython_internal_update_method(self, event) nogil:
+        with gil:
+            self.update(event)
+
+    def update(self, event: list):
+        """
+        Method that is called, when event arrives to related Subscription. Currently (Cython version 0.29.17),
+        abstract methods are not implemented, so this implementation is sort of stub method.
+
+        Parameters
+        ----------
+        event: list
+            List of various data specific to certain event type
+        """
+        warn(Warning('You have not implemented update method in your EventHandler, that is called, when event comes!'))
+
+cdef class DefaultHandler(EventHandler):
+    """
+    The class implements one possible event handler, which is considered as default. This class just stores upcoming
+    events in custom deque with thread lock and has methods to get data as list or as pandas.DataFrame.
+
+    Attributes
+    ----------
+    data_len: int
+        The length of internal DequeWithLock object. Default is 100000.
+    columns: list
+        After attaching listener the field contains one-word descriptors of the values in upcoming event the order
+        coincides.
+    """
+    def __init__(self, data_len: int=100000):
+        super().__init__()
+        self.__data = deque_wl(maxlen=data_len)
+
+    def update(self, event: list):
```

Comment:
Better to use `Iterable` or `Sequence` from typing. In terms of function signature list means exact list type, however we may use here tuple.